api/src/main/java/org/openmrs/module/quarterlyreporting/db/daoimpl/QuarterlyReportingDAOImpl.java

```diff
@@ -3070,7 +3070,7 @@ public boolean isPatientOnProphylaxisOnly(Patient p) {
 		List<Integer> regimenDrugs = new ArrayList<Integer>();
 
 		for (RegimenComponent rc : components) {
-			if (!rc.getDrugOrder().getDiscontinued())
+			if (!rc.getDrugOrder().isActive())

```

Comment:
rc.getDrugOrder().getDiscontinued() was returning true is order was in-active, the opposite of it would mean the order is active which is not what this line is doing